checks/main/main.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"sort"
+
+	"gopkg.in/yaml.v2"
+)
+
+type doc struct {
+	Description string   `yaml:"description"`
+	Remediation []string `yaml:"remediation"`
+}
+
+func main() {
+	yamlFile, err := os.Open("../checks.yaml")
+	if err != nil {
+		panic(err)
+	}
+	defer yamlFile.Close()
+
+	byteValue, err := ioutil.ReadAll(yamlFile)
+	if err != nil {
+		panic(err)
+	}
+	m := make(map[string]map[string]doc)
+	err = yaml.Unmarshal(byteValue, &m)
+	if err != nil {
+		panic(err)
+	}
+	keys := make([]string, 0, len(m["checks"]))
+	for k := range m["checks"] {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
```

Comment:
why sort? we might want to preserve the logical order specified in the yaml

Reply:
If I am looking for docs I would expect things to be sorted so that I can go look for specific items. 

Most of the docs are sorted and that was the reason.